android/build.gradle

```diff
@@ -20,7 +22,7 @@ def safeExtGet(prop, fallback) {
 }
 
 android {
-    compileSdkVersion safeExtGet('compileSdkVersion', 28)
+    compileSdkVersion safeExtGet('compileSdkVersion', 29)
```

Comment:
```suggestion
    compileSdkVersion safeExtGet('compileSdkVersion', 30)
```

we should be able to compile against 30 now?

Reply:
yes, if this configuration works with defaults in your project